### DISCUSSION TITLE
Ender 5 plus Not ready : (

### DISCUSSION THREAD

**Discussion by manuelcyan** [Desuuuu/DGUS-reloaded-Klipper#24] [Q&A] [answered]
hello desuuu
i have installed your fork but i can't get past the klipper image screen ... i have an ender 5 plus stock with a skr 2 smt32f429 and raspberry 4.
In the compilation I chose the card, 8 mhz crystal, communication interface usb and desuuu usart1 ... I have verified gnd 5v Pa9 / Tx2 and pa10 / Rx2.
in the display firmware use the .conf file that formats the display.

Please help

**Comment by manuelcyan** [comment]
![Captura de pantalla 2022-01-10 a las 10 14 43](https://user-images.githubusercontent.com/30378354/148750980-db0991b9-6bc4-427b-a030-b10e95066379.png)

**Comment by Desuuuu** [comment]
That's most likely a communication issue between the board and touchscreen. Your klippy logs could help diagnose the issue further.

**Comment by manuelcyan** [comment, marked as answer]
solution:
Pa09--->Rx2
Pa10--->Tx2
create printer.conf and building and flash mcu
thank

**Reply by napter** [reply]
Can you explain what this means? I'm trying to understand this but what does "Pa09--->Rx2" mean?

Also when you say "create printer.conf " do you mean printer.cfg?

**Comment by manuelcyan** [comment]
now I have a problem with the bltouch...it comes out open in both positions and I already turned the magnet adjustment screw but still without a solution...before it worked perfectly...the bltouch reset screen button comes out busy
